cmd/security-secrets-setup/start_vault.sh

```diff
@@ -32,12 +32,8 @@ listener "tcp" {
               tls_key_file = "'${VAULT_TLS_PATH}'/server.key" 
               tls_perfer_server_cipher_suites = "true"
           } 
-          backend "consul" { 
-              path = "vault/" 
-              address = "edgex-core-consul:8500" 
-              scheme = "http" 
-              redirect_addr = "https://edgex-vault:8200" 
-              cluster_addr = "https://edgex-vault:8201" 
+          backend "file" {
+              path = "vault/file"
```

Comment:
Shouldn't this be `/vault/file?`

Reply:
docker `Vault` doesn't specify the working dir so the default is / and thus both are equivalent in the context of docker vault. and yes, I can change it to /vault/file.

Reply:
The difference is that here it is an **actual path**, vs with consul it was relative to the root of the configuration store.

Reply:
I agree with @bnevis-i . We should be explicit in the name (`/vault/file`) and not depend on implicit implementation.

Reply:
agreed, changed